### DISCUSSION TITLE
Contributing to bevy_ecs_tiled

### DISCUSSION THREAD

**Discussion by stevepryde** [adrien-bon/bevy_ecs_tiled#1] [General]
If you would like to contribute but you're unsure where to start, here is a short wishlist and some notes on how to get started:

- Documentation!
  - docs.rs API documentation and feature introductions
  - A book (github pages)
  - A blog post (I can add a link to it in the README)
- Expose more features from [rs-tiled](https://github.com/mapeditor/rs-tiled)
  - These are the official rust bindings and they are very good.
- Integrate with popular bevy physics crates (please use feature flags)
- Help other users, respond to issues, help review PRs, etc.
  - Maintaining an open source crate can be a lot of work, but it's easier if we do it together
- Contribute to [bevy_ecs_tilemap](https://github.com/StarArawn/bevy_ecs_tilemap) (this crate depends heavily on that one), or anything else in the bevy ecosystem

**Comment by adrien-bon** [comment]
Added some issues with tags to organize a bit the repository:
- [current limitations](https://github.com/adrien-bon/bevy_ecs_tiled/issues?q=is%3Aopen+is%3Aissue+label%3Alimitation) we would like to resolve
- [possible enhancements](https://github.com/adrien-bon/bevy_ecs_tiled/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) we want to have

**Comment by adrien-bon** [comment]
Closing this discussion since there is a dedicated section in the book to address this topic: https://adrien-bon.github.io/bevy_ecs_tiled/misc/contributing.html
